tests/model_serving/model_runtime/vllm_omni/s3/test_vllm_omni_e2e.py

```
@@ -0,0 +1,762 @@
+        vllm_omni_isvc_url: str,
+    ) -> None:
+        """GET /v1/models returns a valid OpenAI-compatible model list.
+
+        Given a deployed vLLM-Omni InferenceService serving Qwen3-Omni-30B-A3B,
+        When a GET request is sent to /v1/models,
+        Then the response has HTTP 200 with "object": "list" and a non-empty "data" array
+        containing at least one entry whose "id" matches the served model name,
+        and each entry carries "object": "model".
+        """
+        base_url = vllm_omni_isvc_url
+        response = requests.get(
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)